spec/factories/books.rb

```diff
@@ -1,8 +1,22 @@
 FactoryBot.define do
   factory :book do
-    title { "MyString" }
-    description { "MyString" }
-    price { "MyString" }
-    is_featured { "MyString" }
+    title { "Book 1" }
+    description { "This is a test book" }
+    price { 1.0 }
+    is_featured { false }
+
+    before(:create) do |book|
+      book.images.attach(
+        io: File.open(Rails.root.join('spec', 'files', 'images', 'book_image.jpg')),
+        filename: 'book_image.jpg',
+        content_type: 'image/jpeg'
+      )
+      tag = Tag.first.blank? ? create_list(:tag, 1) : Tag.first
```

Comment:
`create_list` returns an array, while `Tag.first` returns a single model. Should we make them consistent here?